tests/baselines/reference/commonTypeIntersection.errors.txt

```diff
@@ -0,0 +1,26 @@
+tests/cases/conformance/types/intersection/commonTypeIntersection.ts(2,5): error TS2322: Type '{ __typename?: "TypeTwo"; } & { a: boolean; }' is not assignable to type '{ __typename?: "TypeOne"; } & { a: boolean; }'.
+  Type '{ __typename?: "TypeTwo"; } & { a: boolean; }' is not assignable to type '{ __typename?: "TypeOne"; }'.
+    Types of property '__typename' are incompatible.
+      Type '"TypeTwo"' is not assignable to type '"TypeOne"'.
+tests/cases/conformance/types/intersection/commonTypeIntersection.ts(4,5): error TS2322: Type '{ __typename?: "TypeTwo"; } & string' is not assignable to type '{ __typename?: "TypeOne"; } & string'.
+  Type '{ __typename?: "TypeTwo"; } & string' is not assignable to type '{ __typename?: "TypeOne"; }'.
+    Types of property '__typename' are incompatible.
+      Type '"TypeTwo"' is not assignable to type '"TypeOne"'.
+
+
+==== tests/cases/conformance/types/intersection/commonTypeIntersection.ts (2 errors) ====
+    declare let x1: { __typename?: 'TypeTwo' } & { a: boolean };
+    let y1: { __typename?: 'TypeOne' } & { a: boolean} = x1;  // No error!

```

Comment:
No error?

Reply:
It's a copy & paste from the issue which expressed surprise that no error was issued here. There _should_ be one.

Reply:
I should change the comment; it's "No error!" because the OP was surprised at the lack of error.